etcdctl/ctlv3/command/printer.go

```diff
@@ -163,6 +164,47 @@ func (p *printerUnsupported) DBStatus(snapshot.Status)  { p.p(nil) }
 
 func (p *printerUnsupported) MoveLeader(leader, target uint64, r v3.MoveLeaderResponse) { p.p(nil) }
 
+func makeHeaderTable(h *pb.ResponseHeader) (hdr []string, rows [][]string) {
```

Comment:
Good idea to keep this name generic as it can be reused.  We may want to separate the words in name (here and in the  makeGetTable) instead of keeping at as variable names and keep it consistent with other tables like makeMemberListTable and makeEndpointStatusTable

Reply:
@spzala Can you pls suggest what function names are you proposing? I made it make**Get**Table since it was for get command. Here are functions: makeHeaderTable, makeGetTable, makeFooterTable.

Reply:
@agargi sorry for the confusion with my comment, I was saying that it's good that you have kept the func name generic as makeHeaderTable something that can be reused with other KV commands. The second part of my comment was column names e.g. `RaftTerm` to `Raft Term` (to be more consistent with https://github.com/etcd-io/etcd/blob/master/etcdctl/ctlv3/command/printer.go#L203) but that's a nit. Hope makes sense?

Reply:
@spzala Changed the code and PR'd as per the comment

Reply:
Thanks @agargi